docs/public/deploying-a-devfile-using-odo-on-IBM-Z-and-Power.adoc

```diff
@@ -0,0 +1,93 @@
+== Introduction to devfile
+
+What is a devfile?
+
+A https://redhat-developer.github.io/devfile/[devfile] is a portable file that describes your development environment. It allows reproducing a _portable_ developmental environment without the need of reconfiguration.
+
+With a devfile you can describe:
+
+* Development components such as container definition for build and application runtimes
+* A list of pre-defined commands that can be run
+* Projects to initially clone
+
+odo takes this devfile and transforms it into a workspace of multiple containers running on Kubernetes or OpenShift.
+
+Devfiles are YAML files with a defined https://devfile.github.io/devfile/_attachments/api-reference.html[schema].
+
+== odo and devfile
+
+odo can now create components from devfiles as recorded in registries. odo automatically consults the https://github.com/odo-devfiles/registry[default registry] but users can also add their own registries. Devfiles contribute new component types that users can pull to begin development immediately.
+
+An example deployment scenario:
+
+. `odo create` will consult the recorded devfile registries to offer the user a selection of available component types and pull down the associated `devfile.yaml` file
+. `odo push` parses and then deploys the component in the following order:
+ .. Parses and validates the YAML file
+ .. Deploys the development environment to your Kubernetes or OpenShift cluster
+ .. Synchronizes your source code to the containers
+ .. Executes any prerequisite commands
+
+== Deploying your first devfile on IBM Z & Power
+Since the DefaultDevfileRegistry don't support IBM Z & Power now, you will need to create one secure private devfileregistry firstly. The steps to create new secure private devfileregistry , please check the doc link:https://github.com/openshift/odo/blob/main/docs/public/secure-registry.adoc[secure registry].
```

Comment:
```suggestion
Since the DefaultDevfileRegistry doesn't support IBM Z & Power now, you will need to create a secure private DevfileRegistry first. To create a new secure private DevfileRegistry, please check the doc link:https://github.com/openshift/odo/blob/main/docs/public/secure-registry.adoc[secure registry].
```